### PR TITLE
Release for ECCW-638 (Fathom Analytics) to main

### PR DESCRIPTION
**Pull request by Polynya**
Config exported from Pre-Prod.
Update localgov_geo package to fix dependency on localgov_geo_update module.

### CODE REVIEW

**Review by siliconmeadow — APPROVED** [review]
LGTM

File: .gitlab-ci.yml
Comment (siliconmeadow) [inline .gitlab-ci.yml:1]: @Polynya - this _looks_ like it reverts [this commit](https://github.com/essexcountycouncil/essex-gov-uk-drupal/commit/dcbab060b47f6e27f9c5d2e803f191ce75d4998a). No?

Comment (Polynya) [reply .gitlab-ci.yml:1]: Thanks for checking! I'd pulled changes from main but hadn't committed them. Now there are only changes to Composer and config.

**Review by siliconmeadow — APPROVED** [review]
LGTM

**Pull request merged**